src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -1,31 +1,33 @@
 package com.google.alpollo;
 
-import java.io.IOException;
-
 import com.google.api.gax.rpc.FixedHeaderProvider;
 import com.google.cloud.language.v1.Document;
 import com.google.cloud.language.v1.LanguageServiceClient;
 import com.google.cloud.language.v1.LanguageServiceSettings;
 import com.google.cloud.language.v1.Sentiment;
+import com.google.gson.Gson;
 
 /**
  * Helper class for cleaner code and easier testing.
  */
 
 public class AnalysisHelper {
-  
+  private final Gson gson = new Gson();
+
   /**
    * Based on the lyrics sent, the AI can extract the main "sentiment" of the text.
    * This sentiment has a score, showing the overall positivity of the text, ranging from -1 to 1
    * and a magnitude, representing how strong the sentiment is, ranging from 0 to 1
    */
-  public Sentiment getSentiment(String lyrics) throws IOException{
+  public Sentiment getSentiment(String lyrics) throws Exception {
```

Comment:
I'm not sure that using all kinds of exceptions here is good idea. Specify which ones you want to throw.